Sources/System/FilePath/FilePathString.swift

```diff
@@ -20,6 +20,30 @@ extension FilePath {
     self.init(_platformString: platformString)
   }
 
+  public init(platformString: [CInterop.PlatformChar]) {
```

Comment:
```suggestion
  public init<C: Collection>(platformString: C) where C.Element == CInterop.PlatformChar {
```

Reply:
I'm not sure we want to go generic unless we ultimately want to keep that overload in place. Ideally this would be `@_alwaysEmitIntoClient` so that we can remove it once the language has evolved to eliminate the issue we're mitigating against. That's an interesting point of discussion…

Reply:
Can you write a doc comment that carefully explains the null is expected here? Can you try AEIC? Would a deprecation annotation be possible?

Reply:
For the FilePath ones I'm not sure what to put in the deprecation messages. It's pretty easy for String...